doc/Containerfile.example

```diff
@@ -33,13 +52,17 @@ COPY . .
 RUN \
     ./autogen.sh && \
     ./configure && \
-    make clean all && \
-    make test
+    make clean all
+
+ARG SKIP_TESTS=0
+
+RUN if [ 0 -eq "$SKIP_TESTS" ]; then make test; fi
 
 
 FROM docker.io/library/debian:12
 
 COPY --from=builder /n3n/apps/n3n-edge /n3n-edge
+COPY --from=builder /n3n/apps/n3n-supernode /n3n-supernode
```

Comment:
I didn't get the impression that there is a "standard" port recommended for a supernode, otherwise I'd have put in an `EXPOSE 1234` here, too.